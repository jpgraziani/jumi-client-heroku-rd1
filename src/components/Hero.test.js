import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Hero from './Hero'

describe(`Hero component`, () => {
  it('renders a .Hero by default', () => {
    const wrapper = shallow(<Hero />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})