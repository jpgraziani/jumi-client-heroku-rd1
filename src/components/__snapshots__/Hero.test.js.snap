// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a .Banner by default 1`] = `
<section
  className="footer-container"
>
  <div
    className="nav-footer"
  >
    <Link
      className="footerLink"
      replace={false}
      to="/"
    >
      home
    </Link>
    <Link
      className="footerLink"
      replace={false}
      to="/recipes"
    >
      recipes
    </Link>
  </div>
  <br />
  <div>
    <h6
      className="end-quote"
    >
      exploring desert life on the west coast
    </h6>
  </div>
</section>
`;

exports[`Hero component renders a .Hero by default 1`] = `<header />`;
